pkg/event-manager/drivers/docker_backend.go

```diff
@@ -158,3 +181,18 @@ func (d *dockerBackend) Delete(ctx context.Context, driver *entities.Driver) err
 
 	return err
 }
+
+//GetFreePort return a free port
+func GetFreePort() (int, error) {
```

Comment:
Can you comment this function. I'm not sure how it gets a free port.

Reply:
I also don't know, I just google a function, and it works